ha/setup/ha_setup.py

```diff
@@ -415,15 +415,15 @@ def process(self):
         s3_instances = ConfigCmd.get_s3_instance(machine_id)
         ios_instances = ConfigCmd.get_ios_instance(machine_id)
 
+        self._update_env(node_name, node_type, const.HA_CLUSTER_SOFTWARE, s3_instances, ios_instances)
+        self._fetch_fids()
+        self._update_cluster_manager_config()
+
         # fetch all nodes stonith config
         all_nodes_stonith_config: dict = {}
         if self.get_installation_type().lower() == const.INSTALLATION_TYPE.HW.value.lower():
```

Comment:
stonith_enable = false
if HW:
    all_nodes_stonith_config = ConfigCmd.get_stonith_config()
    stonith_enable = true
elif VM:
    Log.warn("Stonith configuration not available, detected VM env")
else: 
    raise exception

// Now on other place use `stonith_enable ` variable 

// Also configure_stonith is only called in HW then it will either configure stonith or raise error

Reply:
Done fixed.

Reply:
```
// Also configure_stonith is only called in HW then it will either configure stonith or raise error
```
this change is missing

Reply:
addressed review comment. Done.